README.md

```diff
@@ -1,20 +1,54 @@
-# qri
+# [qri](http://qri.io)
 
 [![Qri](https://img.shields.io/badge/made%20by-qri-magenta.svg?style=flat-square)](https://qri.io)
 [![GoDoc](https://godoc.org/github.com/qri-io/qri?status.svg)](http://godoc.org/github.com/qri-io/qri)
 [![License](https://img.shields.io/github/license/qri-io/qri.svg?style=flat-square)](./LICENSE)
 [![Codecov](https://img.shields.io/codecov/c/github/qri-io/qri.svg?style=flat-square)](https://codecov.io/gh/qri-io/qri)
 [![CI](https://img.shields.io/circleci/project/github/qri-io/qri.svg?style=flat-square)](https://circleci.com/gh/qri-io/qri)
 
-Frictionless distributed data.
-qri (“query”) animates data stored on the distributed web to work like a database.
+#### qri is a global dataset version control system (GDVCS) built on the distributed web
+
+Breaking that down:
+
+- **global** so that if *anyone, anywhere* has published work with the same or similar datasets, you can discover it.
+- Specific to **datasets** to ****solve the friction problem.  datasets on qri work with each other by default
```

Comment:
should this be:

**solve**

not ****solve

?

Reply:
ugh, yes.